examples/mongodbatlas_cluster/tenant-upgrade/README.md

```diff
@@ -1,4 +1,7 @@
 # MongoDB Atlas Provider -- Cluster Tenant Upgrade
+
+**WARNING:** `mongodbatlas_cluster` resource and corresponding data sources have been deprecated in favor of `mongodbatlas_advanced_cluster`. In order to learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide).
```

Comment:
Succint phrasing: "To learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide)."

Reply:
thanks, changed